sdk/main/include/MnemonicBIP39.h

```diff
@@ -73,6 +79,27 @@
    */
   static MnemonicBIP39 generate24WordBIP39Mnemonic();
 
+  /**
+   * Generate an ED25519PrivateKey from this Mnemonic using the input passphrase at the specified index.
+   *
+   * @param passphrase The passphrase to use to generate the ED25519PrivateKey.
+   * @param index      The index in the derivation path from which to derive the ED25519PrivateKey.
+   * @return A pointer to the derived ED25519PrivateKey.
+   */
+  [[nodiscard]] std::unique_ptr<ED25519PrivateKey> toStandardEd25519PrivateKey(const std::string& passphrase = "",
+                                                                               int index = 0) const;
+
+  /**
+   * Generate an ECDSAsecp256k1PrivateKey from this Mnemonic using the input passphrase at the specified index.
+   *
+   * @param passphrase The passphrase to use to generate the ECDSAsecp256k1PrivateKey.
+   * @param index      The index in the derivation path from which to derive the ECDSAsecp256k1PrivateKey.
+   * @return A pointer to the derived ECDSAsecp256k1PrivateKey.
+   */
+  [[nodiscard]] std::unique_ptr<ECDSAsecp256k1PrivateKey> toStandardEcdsaSecp256k1PrivateKey(
```

Comment:
The correct function name is `toStandardECDSAsecp256k1PrivateKey`

Reply:
Fixed